### DISCUSSION TITLE
dhaqsoPay light version logo

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#70] [Show and tell]
dhalinyaro waa diyaar logo-da dhaqsoPay light version-keeda.

![image](https://github.com/user-attachments/assets/273b5a62-13fa-4df2-8143-1781ec432427)

**Comment by hayle102** [comment]
Waan ka haynaa Eng

**Comment by Nasrah-muse** [comment]
Wow Masha Allah 🔥
